README.md

```diff
@@ -38,90 +38,132 @@
 </p>
 
 
-# Table of Contents
+**StellarGraph** is a Python library for machine learning on [graphs and networks](https://en.wikipedia.org/wiki/Graph_\(discrete_mathematics\)).
+
+## Table of Contents
    * [Introduction](#introduction)
-   * [Guiding Principles](#guiding-principles)
    * [Getting Started](#getting-started)
+   * [Getting Help](#getting-help)
+   * [Example: GCN](#example-gcn)
+   * [Algorithms](#algorithms)
    * [Installation](#installation)
        * [Install StellarGraph using PyPI](#install-stellargraph-using-pypi)
        * [Install StellarGraph in Anaconda Python](#Install-stellargraph-in-anaconda-python)
        * [Install StellarGraph from Github source](#install-stellargraph-from-github-source)
-   * [Docker Image](#docker-image)
-   * [Running the examples](#running-the-examples)
-       * [Running the examples with docker](#Running-the-examples-with-docker)
-   * [Algorithms](#algorithms)
-   * [Getting Help](#getting-help)
-   * [Discourse Community](#discourse-community)
-   * [CI](#ci)
+       * [Docker Image](#docker-image)
    * [Citing](#citing)
    * [References](#references)
 
 ## Introduction
-**StellarGraph** is a Python library for machine learning on graph-structured (or equivalently, network-structured) data.
 
-Graph-structured data represent entities, e.g., people, as nodes (or equivalently, vertices),
-and relationships between entities, e.g., friendship, as links (or
-equivalently, edges). Nodes and links may have associated attributes such as age, income, and time when
-a friendship was established, etc. StellarGraph supports analysis of both homogeneous networks (with nodes and links of one type) and heterogeneous networks (with more than one type of nodes and/or links).
+The StellarGraph library offers state-of-the-art algorithms for [graph machine learning](https://medium.com/stellargraph/knowing-your-neighbours-machine-learning-on-graphs-9b7c3d0d5896), making it easy to discover patterns and answer questions about graph-structured data. It can solve many machine learning tasks:
 
-The StellarGraph library implements several state-of-the-art algorithms for applying machine learning methods to discover patterns and answer questions using graph-structured data.
-
-The StellarGraph library can be used to solve tasks using graph-structured data, such as:
 - Representation learning for nodes and edges, to be used for visualisation and various downstream machine learning tasks;
 - Classification and attribute inference of nodes or edges;
```

Comment:
should we pre-empt the addition of graph classification here?

Reply:
I'm in two minds, one is that the graph classification works should land soon and so we can just short cut to including, and the other is that people arriving after this lands (if it lands first) to our front page will see this but no evidence of how to do so. Similarly for the dynamic/temporal graph work.

Reply:
Hopefully people will be curious and raise an issue or discourse thread if they can't find it?!

Reply:
#1053 has merged, so I've added it here!